package/suseconnect-keepalive.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Schedule daily SUSEConnect --keepalive calls
+After=network-online.target
+
+[Timer]
+# Run this timer every day at 3am UTC at a randomized 4h delay.
+OnCalendar=daily
+RandomizedDelaySec=24h
```

Comment:
add a `Unit=` section here to ensure that the timer calls the service.

Reply:
That's not really needed since, as stated in the documentation, when not specified, systemd picks the service with the same name as the timer, which is our case.

Reply:
Ah I see.